api/models.py

```diff
@@ -858,7 +834,7 @@ def get_asctime(self):
         )
 
     def get_last_reply_age(self):
-        return self._get_age(self.last_reply_date or self.date)
+        return self.last_reply_date or self.date
```

Comment:
Likewise, we probably want to change the method name, `get_last_reply_date()` probably?